src/components/PropsTable.tsx

```diff
@@ -1,135 +1,194 @@
 
-const Table = styled('table', {
-  borderCollapse: 'collapse',
-  width: '100%'
-})
-const Cell = styled('td', {
-  borderBottom: '1px solid $tonal300',
-  color: '$tonal800',
-  fontSize: '$md',
-  fontFamily: '"Inter"',
-  py: '$3',
-  pr: '$3',
-  textAlign: 'left',
-  verticalAlign: 'middle',
-  variants: {
-    appearance: {
-      heading: {
-        fontSize: '$sm',
-        whiteSpace: 'nowrap',
-        fontWeight: 600
-      }
-    }
-  }
-})
+const columns = ['Prop', 'Type', 'Default', 'Required']
 
 const Empty = () => <Text css={{ color: '$tonal500' }}>-</Text>
 
+const WithTooltip = ({ children, content }) => (
+  <Tooltip>
+    <Tooltip.Trigger>{children}</Tooltip.Trigger>
+    <Tooltip.Content
+      css={{
+        width: '90vw',
+        textAlign: 'center',
+        lineHeight: 1.4,
+        letterSpacing: '0.01em',
+        fontSize: '$sm'
+      }}
+    >
+      {content}
+    </Tooltip.Content>
+  </Tooltip>
+)
+
 const PropType = ({ name, type }) => {
   if (name === 'css') {
     return (
-      <Link href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L1547">
-        <InlineCode>"CSSProperties"</InlineCode>
-      </Link>
+      <WithTooltip content="Override the component styles. Supports tokens, media queries and all stitches features">
+        <Link
+          size="sm"
+          href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L1547"
+        >
+          <InlineCode>CSSProperties</InlineCode>
+        </Link>
+      </WithTooltip>
     )
   }
   if (name === 'as') {
     return (
-      <Link href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L2993">
-        <InlineCode>"JSX.IntrinsicElements"</InlineCode>
-      </Link>
+      <WithTooltip content="Change the component to a different HTML tag or custom component">
+        <Link
+          size="sm"
+          href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L2993"
+        >
+          <InlineCode>JSX.IntrinsicElements</InlineCode>
+        </Link>
+      </WithTooltip>
     )
   }
   if (Array.isArray(type.value)) {
+    const values = type.value
+      .filter(({ value }) => value !== 'undefined')
+      .filter(({ value }) => !value.startsWith('{ [x: string]'))
     return (
       <Flex css={{ gap: '$2', flexWrap: 'wrap' }}>
-        {type.value
-          .filter(({ value }) => value !== 'undefined')
-          .map(({ value }) => (
+        {values.map(({ value }, index) => (
+          <React.Fragment key={value}>
             <InlineCode key={value}>{value}</InlineCode>
-          ))}
+            {index < values.length - 1 && ' | '}
+          </React.Fragment>
+        ))}
       </Flex>
     )
   }
   return <InlineCode>{type.name}</InlineCode>
 }
 
+const TableProps: React.FC<{ css?: CSS; component: ComponentDoc }> = ({
```

Comment:
I get `PropsTable` ... and I get `PropsTableProps` ... but `TableProps`not being the prop type for a table component is confusing imo

Reply:
Ah yeah that's a leftover from my WIP code, you could have seen `Thing` or `Whatever` instead haha.